recipe/meta.yaml

```diff
@@ -1,44 +1,52 @@
+{% set name = "tinycss2" %}
 {% set version = "1.2.1" %}
 
 package:
-  name: tinycss2
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/t/tinycss2/tinycss2-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 8cff3a8f066c2ec677c06dbc7b45619804a6938478d9d73c284b29d14ecb0627
 
 build:
   number: 0
-  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  #[py<37]
 
 requirements:
   host:
-    - flit
-    - python >=3.5
+    - flit-core >=3.2,<4
```

Comment:
please don't use >= in host section.  It should be 'flit-core 3.2' instead.  Version ranges are only for the 'run' section to be used

Reply:
Ahhh, okay, thanks! I thought we only took the pinnings off of python, but that makes sense and I will remember that for future reference. Thank you!